rpcd/playbooks/roles/rpc_maas/templates/client.my.cnf.j2

```diff
@@ -2,5 +2,5 @@
 
 [client]
 host={{ container_address }}
-user={{ galera_root_user }}
-password={{ galera_root_password }}
+user={{ galera_root_user|default('root') }}
+password={{ galera_root_password|default('secrete') }}
```

Comment:
I don't like that this sets a password that would/could be used in a production environment. I think it would be better to simply fail because the value is unset than it is to provide the deployer the ability to go to production with an insecure password.